bears/general/SpaceConsistencyBear.py

```diff
@@ -38,6 +58,12 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        aspect = (self.section.aspects and
+                  self.section.aspects.get('Indentation'))
+        if aspect:
+            use_spaces = (self.section.aspects.get('Indentation').indent_type
```

Comment:
The result of `self.section.aspects.get('Indentation')` is already referenced by `aspect` at this point. No need to re-evaluate ;)

But anyway... This should be superseded by `@map_setting_to_aspect(..., use_spaces=Indentation.indent_type == 'space')`

But... too many "but"s in here :D ... keep it for now (simplified according to the hint above) to keep the tests working you already wrote